src/main/java/edu/cornell/kfs/module/purap/document/IWantDocument.java

```diff
@@ -50,6 +50,8 @@ public class IWantDocument extends FinancialSystemTransactionalDocumentBase impl
     private static final long serialVersionUID = 1L;
 
     private String step;
+    private String presentationMode;
+    private boolean usersRequestingPresentationModeChange;
```

Comment:
I see that these are both non-persisted fields, but I also see that you defined them on both the document itself and the document form. Can you please explain why you need them on both objects and not just on the form? (I see that the existing `step` variable is in both places, but do the other two variables also need to be on both?)